packages/eds-core-react/src/components/Button/Button.docs.mdx

```diff
@@ -101,7 +101,7 @@ The buttons can be grouped by wrapping them with the `Button.Group` component.
 ```tsx
 import { Button } from '@equinor/eds-core-react'
 
-<Button.Group aria-label="button group">
+<Button.Group aria-label="primary">
```

Comment:
Was this supposed to be changed? Doesn't make much sense... 🤔

Reply:
`ButtonGroup` has `role=group`, which is read by VoiceOver. When adding `aria-label` with a word `group` it will read 2 times group. The VoiceOver is overloaded with unnecessary words. 
So i wanted to create a good pattern for naming `groups`. Conclusion was to get rid  of `group` word from `aria-label` because we have readable `grope` from `role`.

Reply:
Aha! I didn't test it with voiceover so that explains it. Maybe something like you did earlier with "example actions" or something like that then? Just thinking how a user might interpret this example and what kind of description `aria-label` should have

Reply:
In `ButtonGroup` docs describing a11y:

> Button.Group has role="group". 

Maybe adding additional description that this role is read by voiceover? 🤔  
And further more in: 

> You should provide an accessible label with aria-label="label", aria-labelledby="id" or <label>.

Advice to test with voiceover while naming the group 🤔

Reply:
In `ButtonGroup` i changed to something like that:
<img width="405" alt="Screenshot 2022-09-21 at 12 10 23" src="https://user-images.githubusercontent.com/105340174/191478078-9850574b-0ef8-4bde-908f-44544c63b0c0.png">